validator_test.go

```diff
@@ -209,3 +209,63 @@ func TestValidatorOverwriteEmailListDirectly(t *testing.T) {
 		t.Error("email added to list should validate")
 	}
 }
+
+func TestValidatorMultipleEmailsMultipleOnlySubDomains(t *testing.T) {
```

Comment:
Wow - I haven't actually looked at this file closely before, these tests haven't been really touched in 5 years and need some TLC.

If you have the bandwidth, can you align your new test cases to the Test Table type tests we have in the rest of the project? Here's a sample: https://github.com/oauth2-proxy/oauth2-proxy/blob/d67d6e3152e2a1c9614906ee9bd9890f8da9a007/providers/provider_default_test.go#L61

Reply:
You can use either ginkgo style in that sample or traditional `t.Assert`, no preference on that. More just want to see any new test cases structured in either test tables or ginkgo.

Reply:
Refactored the tests to use the provided example.